contracts/DCAStrategies/DCAStrategies/DCAStrategiesManagementHandler.sol

```diff
@@ -4,18 +4,62 @@ pragma solidity >=0.8.7 <0.9.0;
 import '../../interfaces/IDCAStrategies.sol';
 
 abstract contract DCAStrategiesManagementHandler is IDCAStrategiesManagementHandler {
+  struct StrategyOwnerAndVersion {
+    address owner;
+    uint96 latestVersion;
```

Comment:
Ok, I just checked our original doc and I realized that this should be `uint16`. We will have some extra space in this struct, but it will make sense when we store positions